system/lib/pthread/library_pthread_stub.c

```diff
@@ -117,6 +117,9 @@ int pthread_key_delete(pthread_key_t key) {
   return 0;
 }
 
+weak_alias(__pthread_key_delete, pthread_key_delete);
+weak_alias(__pthread_key_create, pthread_key_create);
```

Comment:
So this is necessary because `tss_create` et al are built on these pthread functions but expect them to be name with underscores?

Reply:
yes, and these aliases exist already when we build with thread support.. they were just missing from the stubs.

Reply:
Sounds good, can you put that in the commit description?